teachable.rb

```diff
@@ -0,0 +1,27 @@
+module Teachable
```

Comment:
This works, but the naming is weird (a teacher is teachable? I'd expect a student to be teachable, and hopefully teachers are too, but they are usually teaching, not being taught).

Also, both Apprentice and Senior Teachers *are* Teachers.  So this is a perfect case for inheritance.  (Can you say that Teachers and Students are both X for some X?)